src/index.html

```diff
@@ -17,6 +17,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="/src/images/logo.png"
+        alt="/logo.png"
```

Comment:
The `alt` attribute for the image should provide a meaningful description of the image content rather than just the file name. Consider using a descriptive text like 'Company Logo' or 'Moyo Logo' to improve accessibility.